lib/colorls/core.rb

```diff
@@ -1,7 +1,7 @@
 module ColorLS
   class Core
     def initialize(input=nil, all: false, report: false, sort: false, show: false,
-      one_per_line: false, long: false, almost_all: false, tree: false, help: false, colors: [])
+      one_per_line: false, long: false, almost_all: false, tree: false, colors: [])
```

Comment:
> Also I did not understand the message of the helplog, what the "command" part was referring to... something about grep and the README? 😕

Basically, all misc use-cases. Like, `colorls <filename>`, `colorls *` and `colorls <directory> -sd | grep <search>`